push_notification/subscriber/models/subscriber.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+from common.models.base_entity import BaseEntity
+import enum
+
+
+class SubscriberStatus(enum.Enum):
+    ACTIVE = "ACTIVE"
+    INACTIVE = "INACTIVE"
+    DISCARDED = "DISCARDED"
+
+
+class Subscriber(BaseEntity):
+    endpoint = models.CharField("push service url", null=False, blank=False, max_length=255)
```

Comment:
1. We need to add another column to define which website/store this subscriber belongs to. One subscriber can belong to multiple websites.
2. null and blank are False by default, so we don't need to mention it explicitly here

Reply:
@ma-high-ma As onboarding project only have one subscriber-localhost, Shiva and Akash suggested to remove same